packages/backend/src/document/file-response.util.spec.ts

```
@@ -0,0 +1,118 @@
+      'file',
+      'application/zip',
+      'evil\r\nX-Injected: 1',
+    );
+    expect(headers.disposition).not.toContain('\r');
+    expect(headers.disposition).not.toContain('\n');
+  });
+
+  it('percent-encodes non-ascii titles', () => {
+    expect(fileResponseHeaders('file', 'application/zip', '보고서').disposition).toBe(
+      "attachment; filename*=UTF-8''%EB%B3%B4%EA%B3%A0%EC%84%9C",
+    );
```

ESLint

[error] Replace `fileResponseHeaders('file',·'application/zip',·'보고서').disposition).toBe(⏎······"attachment;·filename*=UTF-8''%EB%B3%B4%EA%B3%A0%EC%84%9C",⏎····` with `⏎······fileResponseHeaders('file',·'application/zip',·'보고서').disposition,⏎····).toBe("attachment;·filename*=UTF-8''%EB%B3%B4%EA%B3%A0%EC%84%9C"`

(prettier/prettier)